packages/good-design/src/advanced/customswitch/CustomSwitch.tsx

```diff
@@ -0,0 +1,206 @@
+import React, { useState } from 'react'
+import { Pressable, Icon, Text, ChevronDownIcon, View, Box } from 'native-base'
+
+import FuseIcon from '../../assets/svg/fuse.svg'
+import CeloIcon from '../../assets/svg/celo.svg'
+// import SwitchIcon from ' ../../assets/svg/arrow-swap.svg'
+
+// TODO: make imports from .svg work
+
+const SwitchIcon = () => (
+  <g strokeLinecap="round" strokeLinejoin="round" strokeWidth="1.5">
+    <path d="m5 10.0909h14l-5.8396-5.0909" stroke="#B0B4BB"/>
+    <path d="m19 13.9091h-14l5.8396 5.0909" stroke="#B0B4BB"/>
+  </g>
+)
+
+const IconList: { [key: string]: string; } = {
+  "Fuse": FuseIcon,
+  "Celo": CeloIcon
+}
+
+const show = {
+  button: {
+    display: "flex",
```

Comment:
should be filled in .theme file as theme or default props

Reply:
done